helm-chart/README.rst

```diff
@@ -62,6 +62,29 @@ Most information related to upgrading from one chart version to another is cover
 in the `values changelog file <https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_.
 For upgrades that require some steps other than modifying the values files to be executed, we add some instructions here.
 
+Upgrading to 0.22.0
+*******************
+This version changes the embedded `renku-gitlab` helm chart from `0.6.0` to
+`0.7.0`. This removes the dependency on an external `redis` in the `renku-gitlab`
+`helm` chart and makes `redis` upgradable in the main `renku` `helm` chart. It does
+mean that the embedded `renku-gitlab` `helm` chart now uses `redis` which is run
+within the `gitlab` omnibus container and this is not recommended for production
```

Comment:
> It does mean that the embedded `renku-gitlab` `helm` chart now uses `redis` which is run
within the `gitlab` omnibus container

Presumably this is the _default_ behavior but can still be overridden to use an external redis?